src/overview/components/ResultList.jsx

```diff
@@ -27,6 +27,15 @@ const timeGapToSpaceGap = makeNonlinearTransform({
 })
 
 const ResultList = ({searchResult}) => {
+
+    //if no result found display relavant message
+    if(searchResult.rows.length === 0){
+        return <p className={styles.message}>
```

Comment:
I ll wait for @Treora 's input in this will change it if necessary ! though I completely agree  with you @obsidianart  and yes I am also picky about coding styles at times it's does'nt matter at the early stage but helps a lot latter on.